test/benchmark.py

```diff
@@ -62,15 +62,32 @@ def human_int(value):
     return int(value) * multiplier
 
 
-def create_test_file(path, max_size, verbose=False):
+def create_test_file(path, max_size, verbose=False, chunk_size=65535):
+    if max_size < 1:
+        raise ValueError("max_size must be >= 1")
+    if chunk_size < 1:
+        raise ValueError("chunk_size must be >= 1")
     size = random.randint(1, max_size)
     if verbose:
         print('creating', path, 'size', size)
-    with open(path, 'wb') as f:
-        # TODO: make this more efficient.
-        while size:
-            f.write(b'%c' % random.randint(0, 255))
-            size -= 1
+    
+    size = random.randint(1, max_size)
+    if verbose:
+        print("creating", path, "size", size, "chunk_size", chunk_size)
```

Comment:
This is duplicate code we can remove. Also printing the chunk_size is not necessary as it is only internal information not relevant to the user. 

```suggestion
```